projects/packages/waf/actions.php

```diff
@@ -7,6 +7,8 @@
 
 namespace Automattic\Jetpack\Waf;
 
+use Automattic\Jetpack\Status\Host;
```

Comment:
Should we add the Status package as a new dependency for the package, if we're to use it here?

Reply:
Ah yes, will fix.

Reply:
@jeherve I see we're actually not depending on any other packages besides the changelogger, despite using a few of them in other places (`Connection\Client` or `Modules` in `class-waf-runner.php`). Since this is getting cherry-picked, should we resolve all of these now, or leave it and fix all of them in a follow-up PR? The change does work in testing, presumably because this package is only used in Jetpack the plugin right now, which includes all the referenced packages anyway.

Reply:
> presumably because this package is only used in Jetpack the plugin right now, which includes all the referenced packages anyway.

Yes, that's most likely it. 

I'm okay with fixing all dependencies in a follow-up PR. 👍